c2corg_ui/static/js/auth/authservice.js

```diff
@@ -260,6 +260,19 @@ app.Authentication.prototype.addAuthorizationToHeaders = function(url,
 
 
 /**
+ * @export
+ * @return {string | boolean} token
+ */
+app.Authentication.prototype.getUserToken = function() {
+  if (this.userData) {
+    return this.userData.token;
+  } else {
+    return false;
```

Comment:
Returning a boolean does not really make sense here. You should rather return a string (empty string?) or else you'll get a JS error when trying to concatenate the result of the function in the ``Authorization`` header, lower in this PR.